ci-operator/step-registry/dpu-operator/e2e-tests-workflow/tests/dpu-operator-e2e-tests-workflow-tests-commands.sh

```diff
@@ -1,72 +1,99 @@
 #!/bin/bash
 
+manage_queue() {
+	GET_QUEUE=$(curl -k -s --resolve "$endpoint_resolve" "https://${endpoint}/queue/api/json")
+
+	QUEUE_FILTERED_JOB_NAME=$(echo $GET_QUEUE | jq '.items[] | select(.task.name == "bare-test")')
```

Comment:
bare-test should not be hardcoded. We should have a variable for this that we can replace with the 217 job in one place.